### PR TITLE
[polygon_array_color_histogram, polygon_array_color_likelihood] message filter should have max_queue_size

### PR DESCRIPTION
**Pull request by YoheiKakiuchi**

### CODE REVIEW

File: jsk_perception/src/polygon_array_color_likelihood.cpp [hunk 1]
```diff
@@ -66,16 +66,16 @@ namespace jsk_perception
       sub_reference_ = pnh_->subscribe(
        "input/reference", 1, &PolygonArrayColorLikelihood::referenceCallback, this);
     }
-    sub_polygon_.subscribe(*pnh_, "input/polygons", max_queue_size_);
-    sub_histogram_.subscribe(*pnh_, "input/histograms", max_queue_size_);
+    sub_polygon_.subscribe(*pnh_, "input/polygons", 10);
+    sub_histogram_.subscribe(*pnh_, "input/histograms", 10);
```
Comment (k-okada) [inline jsk_perception/src/polygon_array_color_likelihood.cpp]: is this ok to set 10 at here? should we use max_queue_size at here and introduce new parameter for message filter?

File: jsk_perception/src/polygon_array_color_histogram.cpp [hunk 2]
```diff
@@ -64,9 +64,9 @@ namespace jsk_perception
   void PolygonArrayColorHistogram::subscribe()
   {
     sub_info_ = pnh_->subscribe("input/info", 1, &PolygonArrayColorHistogram::infoCallback, this);
-    sub_polygon_.subscribe(*pnh_, "input", max_queue_size_);
-    sub_image_.subscribe(*pnh_, "input/image", max_queue_size_);
-    async_ = boost::make_shared<message_filters::Synchronizer<ApproximateSyncPolicy> >(100);
+    sub_polygon_.subscribe(*pnh_, "input", 10);
+    sub_image_.subscribe(*pnh_, "input/image", 10);
+    async_ = boost::make_shared<message_filters::Synchronizer<ApproximateSyncPolicy> >(max_queue_size_);
```
Comment (wkentaro) [inline jsk_perception/src/polygon_array_color_histogram.cpp]: The value of queue_size for syncrhonization was 100 before.
So maybe you should also change the default variable of max_queue_size_ to 100. (currently it is 10)

**Comment by YoheiKakiuchi** [comment]
Fix code

**Comment by wkentaro** [comment]
Could you please update the documentation?
- https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/doc/jsk_perception/nodes/polygon_array_color_histogram.md
- https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/doc/jsk_perception/nodes/polygon_array_color_likelihood.md

**Comment by YoheiKakiuchi** [comment]
add documentation

**Pull request merged**